src/main/java/listeners/MainEventListener.java

```diff
@@ -0,0 +1,232 @@
+            case "clear" -> commandHandler = this::clear;
+            default -> commandHandler = this::unknownCommand;
+        }
+        commandHandler.handle(
+                Objects.requireNonNull(event.getMember()),
+                event.getChannel(),
+                server,
+                Arrays.copyOfRange(tokens, 1, tokens.length));
+    }
+
+    private void help(Member member, TextChannel channel, Server server, String[] args) {
+        EmbedBuilder embedBuilder = new EmbedBuilder();
+        embedBuilder.setTitle("Help!");
+        embedBuilder.setDescription("Possible commands:");
+        embedBuilder.setColor(0xE57D25);
+
+        embedBuilder.addField("$queue <topic>", "Add yourself to a queue.", false);
+        embedBuilder.addField("$showqueue <topic>", "Show the people currently in queue.", false);
+        embedBuilder.addField("$showtopics", "List all topics.", false);
+        embedBuilder.addField("$ready <topic> (admin only)", "Retrieve the next person from the queue.", false);
+        embedBuilder.addField("$clear <topic> (admin only)", "Clear the specified queue.", false);
+        embedBuilder.addField("$maketopic <name> (admin only)", "Create a new topic.", false);
+        embedBuilder.addField("$deletetopic <name> (admin only)", "Delete a topic.", false);
+
+        channel.sendMessage(embedBuilder.build()).queue();
+    }
+
+    private void maketopic(Member member, TextChannel channel, Server server, String[] args) {
+        if (!checkAdmin(member, channel)) return;
+
+        Topic topic = new Topic(args[0]);
+        server.createTopic(topic);
+
+        channel.sendMessage(String.format(
+                "%s Topic role \"%s\" has been created.",
+                member.getAsMention(),
+                args[0])).queue();
+    }
+
+    private void deletetopic(Member member, TextChannel channel, Server server, String[] args) {
+        if (!checkAdmin(member, channel)) return;
+
+        Arrays.stream(server.getTopics())                   // loop over all topics...
+                .filter(t -> t.getName().equals(args[0]))   // ...if this topic's name is args[0]...
+                .findFirst()
+                .ifPresent(server::deleteTopic);            // ...delete it
+
+        channel.sendMessage(String.format(
+                "%s Topic role \"%s\" has been deleted.",
+                member.getAsMention(),
+                args[0])).queue();
+    }
+
+    private void showtopics(Member member, TextChannel channel, Server server, String[] args) {
+        String topicList = Arrays.stream(server.getTopics())
+                .map(Topic::getName)
+                .collect(Collectors.joining("\n"));
+
+        channel.sendMessage(String.format(
+                "%s List of topics:\n%s",
+                member.getAsMention(),
+                topicList)).queue();
+    }
+
+    private void queue(Member member, TextChannel channel, Server server, String[] args) {
```

Comment:
A bit of documentation for the method would be great, even just something like:
```java
/** Queues a user for a specific topic. Operates per-server. **/
```
Or we can go full javadoc to be consistent w/ `authbot-hm` -- thoughts?

Reply:
(Adding docs should be in sep issue btw; this PR is a bit large as-is lol)

Reply:
Yep. External documentation is already discussed in #4; should internal documentation like this be a separate issue, or get merged into that one?

Reply:
I'll come back to this

Reply:
Let's do it as part of #4